src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineInstance.java

```diff
@@ -116,7 +116,7 @@ protected void _terminate(TaskListener listener) throws IOException, Interrupted
 
       // If the instance is running, attempt to terminate it. This is an asynch call and we
       // return immediately, hoping for the best.
-      cloud.getClient().terminateInstanceWithStatus(cloud.getProjectId(), zone, name, "RUNNING");
+      cloud.getClient().terminateInstance(cloud.getProjectId(), zone, name);
```

Comment:
Could you expand a bit on this change please.

Reply:
If instance is preemptied its stopped..
And in old code it did not deleted them, they was left as stopped..

Now one delete them..
And it resolves #77

Reply:
If so, let's change the comment part of "if the instance is running".